ui/admin/app/routes/scopes/scope/workers/index.js

```diff
@@ -28,6 +28,16 @@ export default class ScopesScopeWorkersIndexRoute extends Route {
 
   model() {
     const workers = this.modelFor('scopes.scope.workers');
+    workers.forEach((worker) => {
```

Comment:
Do we generally modify the model data? 

I couldn't figure out a good way of calculating this data (I didn't realize I couldn't just call a function from a template). I think otherwise I'd have to make a template helper for this? 

Let me know if there's a better way to accomplish this.

Reply:
No.  I'd suggest adding a getter on the model itself to return the tag count with some provision to cache the value, since it is unlikely to change often.

Reply:
Yes that would make more sense, I think I was tunnel vision'd on the `workers`

Reply:
I don't think it should be `normalizer`, it looks like those cases are dealing still with actual data from the API and just updating it to the correct format?

In this case this is an extra field that I'm calculating, I think having a getter on the model makes sense! I think I was tunnel visioning on the `workers` list and didn't think to just have it on the `worker` model so I'll update this

Reply:
Aha!! that makes way more sense @randallmorey 

Sorry for the confusion @ZedLi !! 

Thanks for the learning 😃

Reply:
Also re: caching the getter, I'm leaning towards not caching as it shouldn't be that computationally expensive, and judging from this [response](https://stackoverflow.com/a/68185081) and the official docs for [cached](https://api.emberjs.com/ember/4.6/functions/@glimmer%2Ftracking/cached) it might be worse. 

With that in mind, it actually looks like our version of ember doesn't have access to the `cached` decorator yet. If we did want to implement caching, do we normally prefer using the polyfill or just doing the more manual method by using the [cache primitives](https://api.emberjs.com/ember/3.28/modules/@glimmer%2Ftracking%2Fprimitives%2Fcache)?